video/out/wayland_common.c

```diff
@@ -2188,6 +2188,8 @@ bool vo_wayland_init(struct vo *vo)
     update_app_id(wl);
     mp_make_wakeup_pipe(wl->wakeup_pipe);
 
+    bool dmabuf_wayland = !strcmp(wl->vo->driver->name, "dmabuf-wayland");
```

Comment:
Could you change this so the value of this is stored in `vo_wayland_state` itself? The driver being used is tied directly to the lifetime of this struct so it's not like it's possible for it to somehow change in the middle of the VO.

Reply:
Done. And since it's a pre-existing issue it's a new commit that comes before the callback fix.